redash/utils/__init__.py

```diff
@@ -81,10 +81,25 @@ def default(self, o):
             return float(o)
         elif isinstance(o, (datetime.timedelta, uuid.UUID)):
             return str(o)
-        elif isinstance(o, (datetime.date, datetime.time)):
+        # See "Date Time String Format" in the ECMA-262 specification.
```

Comment:
In the future please open a separate pull request for such changes, so they can be probably reviewed and discussed. I completely missed this when reviewing #3026. 

My main concern here is line 98, which breaks queries like `select current_time` (on Postgres).